src/Calendar.php

```diff
@@ -110,6 +110,15 @@ public function addEvent(EventInterface $event)
         return $this;
     }
 
+    public function addEvents(array $events)
+    {
+        $this->events = array_filter($events, function($event) {
```

Comment:
Oh yeah. I did not think of that good you catched that.. :) It was though has an initial function to add events.. :) 

I'll work that out :)